portfolio/src/main/java/com/google/Constants.java

```diff
@@ -0,0 +1,19 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google;
+
+public class Constants {
+    private static final String ENTITY_KIND = "listItem";
```

Comment:
This needs to be `public` rather than `private` in order to be accessible from everywhere. If you only want to use it within its own package, you can omit any visibility specifier, which will make it package-visible, but I think that in this case `public` makes the most sense, since it's conceivable that it could be useful from other packages, and it doesn't seem harmful for other packages to use it. You can read more about visibility at [Controlling Access to Members of a Class](https://docs.oracle.com/javase/tutorial/java/javaOO/accesscontrol.html)